packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -135,7 +135,7 @@ class _NestedScrollViewState extends State<NestedScrollView> {
     return new CustomScrollView(
       scrollDirection: widget.scrollDirection,
       reverse: widget.reverse,
-      physics: new ClampingScrollPhysics(parent: widget.physics),
+      physics: widget.physics ?? new ClampingScrollPhysics(),
```

Comment:
What if I want to provide my own class that extends `ClampingScrollPhysics`? Should we require that the `physics` argument be a `ClampingScrollPhysics`?